src/bsp/Bsp.cpp

```diff
@@ -2741,6 +2792,13 @@ int Bsp::add_texture(const char* name, byte* data, int width, int height) {
 		return -1;
 	}
 
+	BSPMIPTEX* oldtex = nullptr;
+
+	if (oldtex = find_embedded_texture(name))

```

Comment:
assignments inside conditions are hard to read (looks like a bug). I think this is more readable:

```
BSPMIPTEX* oldtex = find_embedded_texture(name);

if (oldtex)
```

Reply:
ok!